images/Dockerfile.ubuntu-20-lts-arm-rpi

```diff
@@ -0,0 +1,62 @@
+ARG BASE_IMAGE=ubuntu:20.04
+
+FROM $BASE_IMAGE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common && \
```

Comment:
@sdwilsh thanks for the sugesstions! i totally agree that we should clean this up, but i would prefer to do this as a second step and then for all ubuntu/debian variants

there are even a lot of packages that we should remove additionally, e.g. snapd? or why do we need nano and neovim

in general it would be good to define a list of packages that are needed to bring up a new kairos flavor and keep this to a minimum

Reply:
@sdwilsh you are fine with merging this for now like this and then do the needed changes after you are done with #821 ?

Reply:
Yeah, I can fix this up.  I'll spend some time on mine this weekend, but I'll have very minimal time soon as I have some friends visiting next week.  I'd rather see this get in before the next release than mine :D

Reply:
yeah no problem only want to get this merged and then i can help you on the other one

Reply:
I'm planning to try and get package parity with that one now, and then we can start cutting packages out, which is likely going to be much easier to reason about and verify that things are working correctly.